### DISCUSSION TITLE
请教下 这个文档中数据集的标注是使用什么软件进行标注制作的 ？

### DISCUSSION THREAD

**Discussion by whisky-12** [PaddlePaddle/PaddleOCR#12718] [Q&A]
请教下 这个文档中数据集的标注是使用什么软件进行标注制作的 ？ 
有没有知道的小伙伴
[基于VI-LayoutXLM的发票关键信息抽取] (https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.7/applications/%E5%8F%91%E7%A5%A8%E5%85%B3%E9%94%AE%E4%BF%A1%E6%81%AF%E6%8A%BD%E5%8F%96.md#%E5%9F%BA%E4%BA%8Evi-layoutxlm%E7%9A%84%E5%8F%91%E7%A5%A8%E5%85%B3%E9%94%AE%E4%BF%A1%E6%81%AF%E6%8A%BD%E5%8F%96)

![image](https://github.com/PaddlePaddle/PaddleOCR/assets/79358023/53e2a208-b632-4846-82a1-00ecccca7588)

**Comment by chenhao86** [comment]
PPOCRLable？

**Comment by whisky-12** [comment]
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/79358023/ae924dca-5b69-482d-8521-48a9c1ab7f0a)
数据集的标注json文件如上 。 具体使用那种标注工具 ，怎么表述的 我也不太清楚 ，如有知道的同学 ，感谢多多指点 ，辛苦

**Comment by chenhao86** [comment]
PPOCRLabel ， 文件 >>  导出标记

**Comment by whisky-12** [comment]
> PPOCRLabel ， 文件 >> 导出标记

您好 ，我标记后点击 -> 导出标记 ， 所导出的json文件中 并没有 label标签

**Comment by whisky-12** [comment]
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/79358023/0b203fe5-0dc0-4152-b34d-ce807b9efa49)


> PPOCRLabel ， 文件 >> 导出标记

而且导出的是txt文件 ，转为json文件后 ，可以看出 json对象中缺少 label这个标签
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/79358023/d8c836f5-9799-4102-9884-5c1d95145ada)

**Comment by chenhao86** [comment]
哪就不清楚了，得问大神了。
